repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -23,6 +24,15 @@
 RELEASES = ((7, 5), (7, 6), (7, 7), (7, 8), (8, 0), (8, 1))  # TODO: bad, bad hardcode
```

Comment:
When you're at it, add more releases here.
```suggestion
RELEASES = ((7, 5), (7, 6), (7, 7), (7, 8), (7, 9), (8, 0), (8, 1), (8, 2), (8, 3))  # TODO: bad, bad hardcode
```

Reply:
Thanks, done.